src/index.html

```diff
@@ -1,13 +1,1385 @@
+    <title>KateVR</title>
+    <link
+      rel="icon"
+      href="./images/favicon.svg"
+      type="image/svg+xml"
+    >
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swiper@9/swiper-bundle.min.css">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
+  <body class="body">
+    <header class="header">
+
+      <div class="container header__container">
+        <div class="header__content">
+          <a href="#" class="header__logo logo">
+            <img src="./images/logo.svg" alt="KatVR">
+          </a>
+          <div class="header__lang">
+            <div class="dropdown header__dropdown">
+              <div class="dropdown__input-container">
+                <input
+                  type="text"
+                  class="dropdown__input header__dropdown-input"
+                  id="lang"
+                  readonly
+                >
+                <span class="dropdown__input-icon header__dropdown-icon">
+                  <div class="triangle"></div>
+                </span>
+                <ul class="dropdown__options header__dropdown-options">
+                  <li class="dropdown__option header__dropdown-option">En</li>
+                  <li class="dropdown__option header__dropdown-option">Ar</li>
+                  <li class="dropdown__option header__dropdown-option">Zh</li>
+                  <li class="dropdown__option header__dropdown-option">De</li>
+                  <li class="dropdown__option header__dropdown-option">It</li>
+                  <li class="dropdown__option header__dropdown-option">Ru</li>
+                  <li class="dropdown__option header__dropdown-option">Ua</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+          <div class="header__burger burger">
+            <span class="burger__line"></span>
+          </div>
+          <nav class="header__menu menu">
+            <div class="top-actions header__top-actions">
+              <div class="menu__cross cross-close top-actions__item--right"></div>
+            </div>
+            <ul class="menu__list header__list">
+              <li class="menu__item">
+                <a href="#about" class="menu__link link-menu">About</a>
+              </li>
+              <li class="menu__item">
+                <a href="#specs" class="menu__link link-menu">Tech</a>
+              </li>
+              <li class="menu__item">
+                <a href="#why" class="menu__link link-menu">Benefits</a>
+              </li>
+              <li class="menu__item">
+                <a href="#contacts" class="menu__link link-menu">Contact</a>
+              </li>
+              <li class="menu__item menu__item--desktop-hidden">
+                <a href="#" class="
+                  menu__link
+                  menu__link--has-sub
+                  link-menu"
+                >
+                  Language
+                </a>
```

Comment:
Need to fix code style everywhere
```suggestion
                <a
                  href="#"
                  class="
                    menu__link
                    menu__link--has-sub
                    link-menu"
                >
                  Language
                </a>
```